include/ck/tensor_operation/gpu/device/impl/device_sparse_embeddings_forward_layernorm.hpp

```diff
@@ -24,16 +24,17 @@ template <typename EmbType,
           typename BetaDataType,
           typename AccDataType,
           typename OutType,
+          typename ElementwiseOperation,
```

Comment:
rename elementwise to emb_elementwise
Otherwise, user does not know which tensor are apply by this elementwise
ex: elementwise could also apply on the output of layernorm 
That is why we have a_elementsie, b_elementwise, c_elementwise in GEMM

Reply:
Done